### DISCUSSION TITLE
[ColorPicker] allow empty color `null/''` as value

### DISCUSSION THREAD

**Discussion by renatodeleao** [chakra-ui/zag#1980] [Ideas]
Currently a controlled `ColorPicker` component requires a valid initial color value to "boot". Would you consider supporting a usecase where the initial value is not defined or can this be considered outside of the scope of the color picker since this "part" is usually shown before the actual picker is displayed?

<img width="45" alt="MacOS empty color picker" src="https://github.com/user-attachments/assets/5caaae06-0fea-4d37-a130-807cf9079d35">

### Possible alternative/workaround 
in user land we could simply assume the something like `rgba(0,0,0,0)` as the "empty" value, coerce `null` values to it and perform the necessary ui side-effects

Cheers! ✌️

**Comment by segunadebayo** [comment]
My recommendation is to do this in your userland code. We want to keep things as explicit as possible in the machine.

It’s a good idea nonetheless

**Comment by segunadebayo** [comment]
Closing as duplicate of #2649.
